analyzer-java/pom.xml

```diff
@@ -39,6 +46,11 @@
       <version>5.10.0</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>net.jimblackler.jsonschemafriend</groupId>
```

Comment:
Do we need to document why we chose this JSON Schema library specifically in the report?

Reply:
I think this should at least end up in an ADR. Depending on whether the alternatives are of any interest this could also be explained in the report